pumpkin-config/src/lib.rs

```diff
@@ -169,7 +173,10 @@ impl LoadConfiguration for AdvancedConfiguration {
 
 impl LoadConfiguration for BasicConfiguration {
     fn get_path() -> &'static Path {
-        Path::new("configuration.toml")
+        match env::var("CONFIG") {
```

Comment:
Personally I'd like to have this env var be more descriptive and as such less likely to cause a name collision with out other environment variables.

Possible suggestion would be to change it to `CONFIG_PATH`.

Reply:
Renamed it to CONFIGURATION_PATH to match filename and be consistent with FEATURES_PATH.